cfme/tests/automate/test_namespace.py

```diff
@@ -94,3 +95,29 @@ def test_duplicate_namespace_disallowed(request, parent_namespace):
         parent_namespace.namespaces.create(
             name=ns.name,
             description=ns.description)
+
+
+@pytest.mark.tier(2)
+def test_namespace_name_wrong(domain):
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/60h
+        tags: automate
+        testSteps:
+            1. Navigate to Automation> Automate> Explorer
+            2. Try to create namespace with name `Dummy Domain` (I put space which is invalid)
+        expectedResults:
+            1.
+            2. Should give proper flash message
+
+    Bugzilla:
+        1650071
+    """
+    view = navigate_to(domain.namespaces, 'Add')
+    view.name.fill('Dummy Namespace')
+    view.add_button.click()
+    view.flash.assert_message('Name may contain only alphanumeric and _ . - $ characters')
```

Comment:
Same comment as above.

Reply:
Same as above , please create domain using collection then just check for flash message( by creating view) and  assert namespace not exist